cmd/secrets/list.go

```diff
@@ -0,0 +1,31 @@
+package secrets
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	. "github.com/tensorleap/leap-cli/pkg/api"
+	"github.com/tensorleap/leap-cli/pkg/auth"
+)
+
+func init() {
+	RootCommand.AddCommand(&cobra.Command{
+		Use:   "list",
+		Short: "List available secrets",
+		Long:  `List available secrets`,
```

Comment:
```suggestion
		Short: "List secrets",
		Long:  `List secrets`,
```

Either that or add available for the manage secrets api description